### ISSUE TITLE
it exclude empty field value from req. body

### ISSUE THREAD

**Issue by martingg88** [balderdashy/skipper#66]
it exclude empty field value from req. body. any idea?
how to make sure it include all the submitted field value including empty value as well

**Comment by martingg88** [comment]
any update on this?

**Comment by radkodinev** [comment]
@martingg88 This is fixed in https://github.com/balderdashy/skipper/pull/61 which is already merged. However, no new version of skipper has been released yet.

**Comment by martingg88** [comment]
cools. hope it can merge with sails library too soon. thanks.

**Comment by tjwebb** [comment]
@sgress454 @mikermcneil can we publish to npm?

**Comment by sgress454** [comment]
Done!

**Comment by sgress454** [comment]
Ok, spoke too soon, I don't have perms on this one.  @mikermcneil, it's up to you.

**Comment by sailsbot** [comment]
Thanks for posting, @martingg88.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
